app/src/main/res/values/dimens.xml

```diff
@@ -166,5 +166,7 @@
     <dimen name="key_gear_item_detail_photos_height">480dp</dimen>
     <dimen name="hedvig_edit_text_padding">14dp</dimen>
     <dimen name="item_detail_gradient_size">300dp</dimen>
+    <dimen name="market_item_padding">18dp</dimen>
```

Comment:
These margins are a bit odd, are they from the designs? 🤔

Reply:
yea, this is wrong, i got the padding from the wrong element